.github/workflows/ci-cd.yml

```
@@ -0,0 +1,404 @@
+            if [ "${{ github.event_name }}" == "pull_request" ]; then
+              BASE_SHA="${{ github.event.pull_request.base.sha }}"
+              HEAD_SHA="${{ github.event.pull_request.head.sha }}"
+            else
+              BASE_SHA="${{ github.event.before }}"
+              HEAD_SHA="${{ github.sha }}"
+            fi
+          
+            # 변경된 서비스 디렉토리 찾기
+            CHANGED_SERVICES=$(git diff --name-only $BASE_SHA $HEAD_SHA | grep '^service/' | cut -d'/' -f2 | sort -u | tr '\n' ' ')
+            DETECTION_METHOD="git_diff"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)